### PR TITLE
Move away from Fedora 31

### PR DESCRIPTION
**Pull request by marusak**
TODO:

 * [x] image-refresh fedora-32
- [x] Introduce new contexts in separate PR so we can test it here (#1195)
- [x] Closes line when picking the new image
- [x] Drop the manual contexts in appropriate commits (blocks at #1195)

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-1184-20200829-081212/

**Comment by cockpituous** [comment]
image-refresh fedora-32 done: https://github.com/cockpituous/bots/commits/image-refresh-fedora-32-20200829-083737

File: task/testmap.py [hunk 1]
```diff
@@ -27,9 +27,9 @@
     'cockpit-project/cockpit': {
         'master': [
             'fedora-31/container-bastion',
-            'fedora-31/selenium-firefox',
-            'fedora-31/selenium-chrome',
-            'fedora-31/selenium-edge',
+            'fedora-32/selenium-firefox',
+            'fedora-32/selenium-chrome',
+            'fedora-32/selenium-edge',
```
Comment (martinpitt) [inline task/testmap.py:32]: After #1195 this now needs to be adjusted to remove the contexts from `_manual`. Technically not a blocker though, that cleanup could be a follow-up.

**Review by martinpitt — COMMENTED** [review]
Thanks, looks good! Now needs to trigger the appropriate tests.

File: task/testmap.py (hunk 1 above)
Comment (marusak) [reply task/testmap.py:32]: right, but wanted to do it in this PR. Just pushed that and now spinning the tests.

**Comment by marusak** [comment]
- [ ] bastion fails with: [log](https://logs.cockpit-project.org/logs/pull-1184-20200829-192639-47c297f9-fedora-32-container-bastion-cockpit-project-cockpit/log.html)

```
error: Failed dependencies:
	systemd is needed by cockpit-ws-226.28.g9c6911c00-1.wip.fc32.x86_64
	systemd >= 235 is needed by cockpit-ws-226.28.g9c6911c00-1.wip.fc32.x86_64
Error: error building at STEP "RUN /container/install.sh": error while running runtime: exit status 1
```

- [ ] In selenium tests I see: [log](https://logs.cockpit-project.org/logs/pull-1184-20200829-192624-47c297f9-fedora-32-selenium-chrome-cockpit-project-cockpit-rhel-7.9/log.html)

```
DEBUG: cockpit-ws:ERROR:src/ws/test-kerberos.c:301:test_authenticate: assertion failed (GSS_S_CONTINUE_NEEDED == status): (1 == 851968): Unspecified GSS failure.  Minor code may provide more information (Server krbtgt/COCKPIT.LAN@COCKPIT.MOCK not found in Kerberos database)
```
but this is in image-prepare, so I don't think it is related to it being selenium, rather than just unstable test.

- [ ] selenium fails with `module time has not attribute clock` [log](https://logs.cockpit-project.org/logs/pull-1184-20200829-192613-47c297f9-fedora-32-selenium-edge-cockpit-project-cockpit/log.html#1)

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-1184-20200831-072828/

**Comment by martinpitt** [comment]
@marusak: When you force-push again to add the new image, can you please fix the "Needs though `systemd` as `fedora:32` image stopped shipping that." commit message? It probably meant to be "Needs to add `systemd` to ..."?

**Comment by cockpituous** [comment]
image-refresh fedora-32 done: https://github.com/cockpituous/bots/commits/image-refresh-fedora-32-20200831-075656

**Comment by martinpitt** [comment]
@marusak: The rhel-7.9 failures look like the branch needs backporting of https://github.com/cockpit-project/cockpit/commit/8eed6e73b5db8010dc33337458f2062e930c44f0

**Review by martinpitt — APPROVED** [review]
Looks great now, thanks! As soon as https://github.com/cockpit-project/cockpit/pull/14531 lands, we can retry this and then land.

**Pull request merged**